src/base/bittorrent/torrentinfo.h

```diff
@@ -95,13 +95,12 @@ namespace BitTorrent
 
         bool matchesInfoHash(const InfoHash &otherInfoHash) const;
 
+        // returns file index or -1 if fileName is not found
+        int fileIndex(const Path &filePath) const;
```

Comment:
I changed this method from private to public. We need the file index in order to fetch each file's size. In my testing `TorrentInfo::filePaths()` returns the files in order, starting from the 0th index, but I figured it'd be better to not implicitly rely on this.

Reply:
See https://github.com/qbittorrent/qBittorrent/pull/21015#discussion_r1661409760